src/uniprotkb/components/statistics/StatisticsPage.tsx

```diff
@@ -0,0 +1,915 @@
+  reviewed,
+  noTitle,
+  caption,
+}: StatsTableProps) => {
+  const hasDescription = category.items.some((item) => item.description);
+  const hasOnlyEntryCounts = category.items.every(
+    (item) => item.count === item.entryCount
+  );
+  // Exceptions
+  const hasEntryCount = category.categoryName !== 'TOTAL_ORGANISM';
+  const hasPercent =
+    category.items.length > 1 && category.categoryName !== 'TOP_ORGANISM';
+  let rows = category.items;
+  if (category.categoryName === 'SEQUENCE_COUNT') {
+    rows = Array.from(category.items).sort((a, b) => +a.name - +b.name);
+  }
+  if (sortByCount.has(category.categoryName)) {
+    rows = Array.from(category.items).sort((a, b) => b.count - a.count);
+  }
+
+  return (
+    <section>
+      {!noTitle && (
+        <h3>
+          {category.label} (UniProtKB {reviewed ? '' : 'un'}reviewed)
+        </h3>
+      )}
+      <table>
+        {caption && <caption>{caption}</caption>}
+        <thead>
+          <tr>
+            <th>Name</th>
+            {!hasOnlyEntryCounts && <th>Count</th>}
+            {hasPercent && !hasOnlyEntryCounts && <th>Percent</th>}
+            {hasEntryCount && <th>Entry count</th>}
+            {hasPercent && hasOnlyEntryCounts && <th>Percent</th>}
+            {/* {!hasOnlyEntryCounts && <th>Per-entry average</th>} */}
+            {hasDescription && <th>Description</th>}
+          </tr>
+        </thead>
+        <tbody>
+          {rows.map((row) => {
+            const percent =
+              hasPercent &&
+              ((row.count / category.totalCount) * 100).toFixed(2);
+            return (
+              <tr key={row.name}>
+                {/* Name */}
+                <td>{row.label || row.name}</td>
+                {/* Count */}
+                {!hasOnlyEntryCounts && (
+                  <td className={styles.end}>
+                    <LongNumber>{row.count}</LongNumber>
+                  </td>
+                )}
+                {/* Percent */}
+                {hasPercent && !hasOnlyEntryCounts && (
+                  <td className={styles.end}>
+                    {percent === '0.00' ? '<0.01' : percent}%
+                  </td>
+                )}
+                {/* Entry count */}
+                {hasEntryCount && (
+                  <td className={styles.end}>
+                    <LongNumber>{row.entryCount}</LongNumber>
+                  </td>
+                )}
+                {/* Percent */}
+                {hasPercent && hasOnlyEntryCounts && (
+                  <td className={styles.end}>
+                    {percent === '0.00' ? '<0.01' : percent}%
+                  </td>
+                )}
+                {/* Per-entry average */}
+                {/* WRONG: This needs to be divided by number of entries in data release */}
+                {/* {!hasOnlyEntryCounts && (
+                <td className={styles.end}>
+                  {(row.count / row.entryCount).toFixed(2)}
+                </td>
+              )} */}
```

Comment:
Is this to be fixed as part of this PR?

Reply:
We can address this in the statistics page PR specifically